packages/@okta/vuepress-site/docs/reference/okta-expression-language/index.md

```diff
@@ -236,6 +236,8 @@ The `user.getGroups` function also supports the `.![name]` collection projection
 
 #### Group-claims only functions
 
+> **Note:** Okta recommends that you use the `user.getGroups` function with collection projections instead. See [Collection projections](/docs/reference/okta-expression-language-in-identity-engine/#collection-projections).
```

Comment:
Great callout, could we get this to also link to the wider federated claims docs? Linking to projection is certainly going to be necessary, but it only helps understand how to build an expression, not where to use them. Susan to help guide on best option there?